comps/animation/wav2lip/dependency/check_wav2lip_server.py

```diff
@@ -10,9 +10,9 @@
 outfile = os.environ.get("OUTFILE")
 
 # Read the JSON file
-with open("comps/animation/wav2lip/assets/audio/sample_question.json", "r") as file:
+with open("comps/animation/wav2lip/assets/audio/sample_whoareyou.json", "r") as file:
     data = json.load(file)
 
-inputs = {"audio": data["byte_str"]}
+inputs = {"audio": data["byte_str"], "max_tokens": 64}
```

Comment:
Where is "max_tokens" used?

Reply:
It's used in here in the gateway: 
https://github.com/opea-project/GenAIComps/blob/9fec2262613e6db71d617cacf321f84d6971dcc6/comps/cores/mega/gateway.py#L944C9-L952C10